packages/appsync-emulator-serverless/lambdaRunner.js

```diff
@@ -1,39 +1,39 @@
 const log = require('logdown')('appsync-emulator:lambdaRunner');
 
+const sendOutput = output => {
+  process.send({ type: 'success', output }, process.exit);
+};
 const sendErr = err => {
-  process.send({ type: 'error', error: err }, () => {
-    process.exit();
-  });
+  process.send({ type: 'error', error: err.message ? err.message : err }, process.exit);
 };
 
-process.once('message', ({ module, handlerPath, handlerMethod, payload }) => {
-  try {
-    log.info('load', module);
-    // eslint-disable-next-line
-    const handlerModule = require(module);
-    if (!handlerModule[handlerMethod]) {
-      sendErr(
-        new Error(
+process.once(
+  'message',
+  async ({ module, handlerPath, handlerMethod, payload }) => {
+    try {
+      log.info('load', module);
+      // eslint-disable-next-line
+      const handlerModule = require(module);
+      if (!handlerModule[handlerMethod]) {
+        throw new Error(
```

Comment:
I think it would be appropriate to keep `sendErr` here so the child process can exit gracefully.

Reply:
`try-catch` can handle it :)